src/bubbleSort.js

```diff
@@ -0,0 +1,26 @@
+export default function bubbleSort(array){
+  var sortingHat = [];
+  if(Array.isArray(array)) {
+    for(var i = 0; i<array.length; i++){
+      if(isNaN(array[i]) === false){
+        sortingHat.push(array[i]);
+      } else {
+        return 'Please enter an array of numbers';
+      }
+    }
+  } else {
+    return 'Please enter an array of numbers';
+  }
+  sortingHat.forEach(function repeater(element){
+    for(var x = 0; x<sortingHat.length; x++){
+      while(sortingHat[x] > sortingHat[x+1]){
+        if(sortingHat[x] > sortingHat[x+1]){
```

Comment:
The while loop is already checking if `sortingHat[x] > sortingHat[x+1]`- the if block here isn't necessary.